src/components/Footer/Footer.tsx

```diff
@@ -40,25 +40,25 @@ class Footer extends Component<Props, State> {
   
   handleUIThemeChange = checked => {
     const { settingsSetUITheme } = this.props
-    const uiTheme = checked ? 'dark' : 'light'
+    const uiTheme = checked ? constants.core.dark : constants.core.light
```

Comment:
I know I started the `core` variable, but it's so generic, if you can group any of the values in it into more specifically named variables, please do. You can put `dark` and `light` in the attributes variable for now.